examples/experimental/trial/unets_tf_keras/const.yaml

```diff
@@ -0,0 +1,23 @@
+description: unets_tf_keras_const
+data:
+  train_url: http://download.tensorflow.org/data/iris_training.csv 
+  test_url: http://download.tensorflow.org/data/iris_test.csv
```

Comment:
Are these necessary?

Reply:
@katport this is for if we choose to use our own datasets, right? If the plan is to merge this into master as a trial example before the City of Hope demo, should we keep this or not?

Reply:
these should be removed - it's leftover from the iris example. 

@naren-determined we try to minimize the amount of extra code to examples. When we add new data we can redesign the const.yaml.